src/rules/function-name-case/README.md

```diff
@@ -97,3 +97,125 @@ a {
   background: -WEBKIT-RADIAL-GRADIENT(red, green, blue);
 }
 ```
+
+## Optional options
+
+### `ignoreFunctions: ["/regex/", "non-regex"]`
+
+Ignore case of function names.
+
+For example, with `"lower"`, given:
+
+```js
+["SOME-FUNCTION", "/^get.*$/"]
+```
+
+The following patterns are considered warnings:
+
+```css
+a {
+  color: sOmE-FuNcTiOn();
+}
+```
+
+```css
+a {
+  color: OTHER-SOME-FUNCTION();
+}
+```
+
+```css
+a {
+  color: GetColor();
+}
+```
+
+```css
+a {
+  color: GET_COLOR();
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+a {
+  display: some-function();
```

Comment:
> or the example that caused my confusion removed.

Lets remove the `display: some-function();` example from the accepted patterns list. 

Even though that behaviour makes sense to me, I think it's needlessly confusing to include it in the example. So, shall we just show that `ignoreFuctions: ["FUNCTION-NAME"]` accepts `a { color: FUNCTION-NAME(x) }` but rejects `a { color: FuNcTiOn-NaMe(x)`